docs/pages/guides/splash-screens.mdx

```diff
@@ -136,7 +136,7 @@ Depending on the `resizeMode` you will get the following behavior:
 
 ### Bare workflow apps
 
-If your app does not use [Expo Prebuild](/workflow/prebuild) (formerly the _managed workflow_) to generate the native `ios` and `android` directories, then changes in the [Expo config][expo-config] will have no effect. To setup and customize your splash screen manually, refer to [this guide](https://github.com/expo/expo/tree/main/packages/expo-splash-screen#-installation-in-bare-react-native-projects).
+If your app does not use [Expo Prebuild](/workflow/prebuild) (formerly the _managed workflow_) to generate the native `ios` and `android` directories, then changes in the [app config][expo-config] will have no effect. To setup and customize your splash screen manually, refer to [this guide](https://github.com/expo/expo/tree/main/packages/expo-splash-screen#-installation-in-bare-react-native-projects).
```

Comment:
```suggestion
If your app does not use [Expo Prebuild](/workflow/prebuild) (formerly the _managed workflow_) to generate the native `android` and `ios` directories, then changes in the [app config][expo-config] will have no effect. To setup and customize your splash screen manually, refer to [this guide](https://github.com/expo/expo/tree/main/packages/expo-splash-screen#-installation-in-bare-react-native-projects).
```

Not related to the PR changes, but let's correct the order since we edit that line anyway. 🙂